v23.1/cockroach-node.md

```diff
@@ -117,10 +117,11 @@ Flag | Description
 `--stats` | Show node disk usage details.
 `--timeout` | Set the duration of time that the subcommand is allowed to run before it returns an error and prints partial information. The timeout is specified with a suffix of `s` for seconds, `m` for minutes, and `h` for hours. If this flag is not set, the subcommand may hang.
 
-The `node decommission` subcommand also supports the following general flags:
+The `node decommission` subcommand also supports the following general flags.  For more information, see `cockroach node decommission --help`.
 
 Flag | Description
 -----|------------
+`--checks` | <a name="decommission-checks"></a> Whether to perform a set of "decommissioning pre-flight checks". Possible values: `enabled`, `strict`, or `skip`. If `enabled`, CockroachDB will check if a node can successfully complete decommissioning given the current state of the cluster. If errors are detected that would result in the inability to complete node decommissioning, they will be printed to `STDERR` and the command will exit *without attempting to perform node decommissioning*. For more information, see [Remove nodes](node-shutdown.html?filters=decommission#remove-nodes).<br/><br/>**Default:** `enabled`
 `--wait` | When to return to the client. Possible values: `all`, `none`.<br><br>If `all`, the command returns to the client only after all replicas on all specified nodes have been transferred to other nodes. If any specified nodes are offline, the command will not return to the client until those nodes are back online.<br><br>If `none`, the command does not wait for the decommissioning process to complete; it returns to the client after starting the decommissioning process on all specified nodes that are online. Any specified nodes that are offline will automatically be marked as decommissioning; if they come back online, the cluster will recognize this status and will not rebalance data to the nodes.<br><br>**Default:** `all`
```

Comment:
I'm not sure if it's worth it or not to document the `--dry-run` flag here.  Perhaps that flag is self-explanatory?  In any case, when `cockroach node decommission <nodeID> --dry-run` is executed, it simply runs the checks, prints the status, and exits.